efabless_tool.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+from ast import arg
```

Comment:
what is this for? is it used anywhere?

Reply:
Not sure why I included it. It is not used anywhere in my changes. I will remove it.